docs/usage/editing-paths.md

```diff
@@ -0,0 +1,78 @@
+Choreo uses trajectory optimization find the optimal route through waypoints. Additionally, Choreo allows fine-grained customization while still maintaining concrete stability. This allows for smoother paths and better odometry.
```

Comment:
It might be nice to mention that you're using "optimization" in the mathematical sense, since many FRC programmers won't have ever heard it used this way.

For example,
```suggestion
Choreo finds the [mathematically optimal](https://en.wikipedia.org/wiki/Mathematical_optimization) trajectory that passes through your waypoints, subject to your drivetrain's constraints. It also allows fine-grained customization while still maintaining concrete stability. This allows for smoother paths and better odometry.
```